src/js/base/js-numbers.js

```diff
@@ -550,96 +534,96 @@ define(function() {
     var deltaIsRough = isRoughnum(delta)
     var argNumsAreRough = isRoughnum(computedValue) || isRoughnum(trueValue)
 
-    var ratCv = isRoughnum(computedValue) ? computedValue.toRational() : computedValue
-    var ratTv = isRoughnum(trueValue) ? trueValue.toRational() : trueValue
+    var ratCv = isRoughnum(computedValue) ? computedValue.toRational(errbacks) : computedValue
+    var ratTv = isRoughnum(trueValue) ? trueValue.toRational(errbacks) : trueValue
 
-    var ratDelta = isRoughnum(delta) ? delta.toRational(): delta
+    var ratDelta = isRoughnum(delta) ? delta.toRational(errbacks): delta
 
-    var err = abs(subtract(ratCv, ratTv))
+    var err = abs(subtract(ratCv, ratTv, errbacks), errbacks)
 
-    if (lessThanOrEqual(ratDelta, 1)) {
-      var absDelta = multiply(ratDelta, abs(ratTv))
+    if (lessThanOrEqual(ratDelta, 1, errbacks)) {
+      var absDelta = multiply(ratDelta, abs(ratTv, errbacks), errbacks)
       if (deltaIsRough && toRoughnum(absDelta).n === Number.MIN_VALUE) {
         if (argNumsAreRough && Math.abs(toRoughnum(err).n) === Number.MIN_VALUE) {
-          throwRuntimeError('roughnum tolerance too small for meaningful comparison, ' +
+          errbacks.throwRelToleranceError('roughnum tolerance too small for meaningful comparison, ' +
                             computedValue + ' ' + trueValue + ' ' + delta)
         }
       }
 
-      return lessThanOrEqual(err, absDelta)
+      return lessThanOrEqual(err, absDelta, errbacks)
     } else {
-      var errRatio = divide(err, abs(ratTv))
+      var errRatio = divide(err, abs(ratTv, errbacks), errbacks)
 
       if (deltaIsRough && delta.n === Number.MIN_VALUE) {
         if (argNumsAreRough && Math.abs(toRoughnum(errRatio).n) === Number.MIN_VALUE) {
-          throwRuntimeError('roughnum tolerance too small for meaningful comparison, ' +
+          errbacks.throwRelToleranceError('roughnum tolerance too small for meaningful comparison, ' +
```

Comment:
I could not figure out how to write a test that would reach this line.  The `toRational` conversions and back aren't quite the identity, so it's hard to probe for exactly these values.